pkg/kudoctl/cmd/get.go

```diff
@@ -5,15 +5,23 @@ import (
 	"github.com/spf13/cobra"
 )
 
-// newGetCmd creates a new command that lists instances
+// newGetCmd creates a command that lists the instances in the cluster
 func newGetCmd() *cobra.Command {
-	newCmd := &cobra.Command{
-		Use:   "get",
-		Short: "-> Show all available instances.",
-		Long:  `The get command has subcommands to show all available instances.`,
+	options := get.DefaultOptions
+	getCmd := &cobra.Command{
+		Use:   "get instances",
+		Short: "Gets all available instances.",
+		Long: `
+	# Get all available instances
+	kudoctl get instances`,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			// Prior to command execution we parse and validate passed parameters
```

Comment:
more copy-paste that does not make sense here

Reply:
yay! done!